lib/network/tsnetwork_bwlimit.c

```diff
@@ -199,9 +199,9 @@ network_bwlimit_get(int op, size_t * len)
 
 	/* Return the appropriate value. */
 	if (op == NETWORK_OP_READ)
-		*len = limit_read.bucket;
+		*len = (size_t)(limit_read.bucket + 0.5);
```

Comment:
Just stick to truncation.  You can't actually get large enough values to have floating-point roundoff, and even if you did it wouldn't result in non-integer values...